File: lecture0-30/lecture002/digits.cpp
#include<iostream>
using namespace std;

int main()
{
    int a;
    cout<<"Enter a no to count no of digits : ";
    cin>>a;
    int digits=0;
    while(a!=0){
        a/=10;
        digits++;
    }
    cout<<"no of digits are "<<digits;
}

File: lecture002/digits.java
import java.util.Scanner;

public class digits {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        System.out.print("Enter a number to count no of digits : ");
        int n = scn.nextInt();
        int digit = 0;
        while (n != 0) {
            n /= 10;
            digit++;
        }
        System.out.print("no of digits in this no is " + digit);

    }
}